_includes/layouts/base.njk

```diff
@@ -5,11 +5,28 @@
 
 	<body>
 
-		<header>
-			<em>
-				<a href="{{ '/' | url }}">{{ metadata.title }}</a>
-			</em>
-			{% include "components/nav.njk" %}
+		<header role="banner" class="header-main" id="main-navigation">
+			<div class="container container--wide container--flex">
+				<div class="header-main__logo">
+					<a href="{{ '/' | url }}">
+						<img src="/static/img/nlc-full-logoX600.png" alt="{{ metadata.title }}" />
+					</a>
+				</div>
+				<div class="header-main__nav">
+					{% include "components/nav.njk" %}
+				</div>
+				<button class="header-main__menu js-closed" aria-expanded="false" href="#main-navigation">
+					<a class="css-nav css-nav--open" href="#main-navigation" aria-label="Open menu without javascript">Open</a>
```

Comment:
I think these don't have to mention javascript to the users

Reply:
Amended.